/* Copyright (C) 2025 Interactive Brokers LLC. All rights reserved. This code is subject to the terms
 * and conditions of the IB API Non-Commercial License or the IB API Commercial License, as applicable. */

syntax = "proto3";

package protobuf;

option java_outer_classname = "HistoricalDataBarProto";
option java_package = "com.ib.client.protobuf";
option csharp_namespace = "IBApi.protobuf";

message HistoricalDataBar {
	optional string date = 1;
    optional double open = 2;
    optional double high = 3;
    optional double low = 4;
    optional double close = 5;
	optional string volume = 6;
	optional string WAP = 7;
    optional int32 barCount = 8;
}
